fix(changelog): add packer category + entries (#844)
scaleway/docs-content: changelog/packer/semptember2022/2022-09-05-packer-added-optional-cleanup-step.mdx

---
title: Optional step to clean up machine-specific data
status: added
author:
    fullname: 'Join the #packer channel on Slack.'
    url: 'https://slack.scaleway.com/'
date: 2022-09-05
category: managed-services
product: packer
---

> We have added an optional step to remove information that is specific to Instances used to build an image, such as the machine ID.

Author: ```suggestion
We have added an optional step to remove information related to the Instance used to build the image, such as its machine ID.
```
